spinalcordtoolbox/download.py

```diff
@@ -375,13 +376,25 @@ def install_named_dataset(dataset_name, dest_folder=None, keep=False):
 
 def list_datasets():
     """
-    :returns: A table containing listing the downloadable datasets
+    :returns: A table listing the downloadable datasets
     :rtype: str
     """
+    color = {True: 'LightGreen', False: 'LightRed'}
     table = f"{'DATASET NAME':<30s}{'TYPE':<20s}\n"
     table += f"{'-' * 50}\n"
-    sortedDatasets = sorted(DATASET_DICT, key=lambda k: DATASET_DICT[k]['download_type'])
+    sortedDatasets = sorted(DATASET_DICT,
+                            key=lambda k: DATASET_DICT[k]['download_type'])
```

Comment:
Slight nitpick: In the dataset ordering, currently we sort just by download type (and not name), so PAM50 and PAM50_normalized_metrics aren't beside one another.

At first, I thought to sort by name within categories:

```suggestion
    sortedDatasets = sorted(DATASET_DICT,
                            key=lambda k: DATASET_DICT[k]['download_type'] + k)
```

However, this un-sorts the list of course datasets, since their years (`17`, `18`, `19`, etc.) come after their names. But, maybe this is okay, given that we plan to tackle https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4009 anyway.